packages/admin/cms-admin/src/form/file/FileField.tsx

```diff
@@ -20,6 +24,75 @@ const FileField = ({ buttonText, input, allowedMimetypes }: FileFieldProps) => {
     const [chooseFileDialogOpen, setChooseFileDialogOpen] = useState<boolean>(false);
     const client = useApolloClient();
 
+    const [anchorEl, setAnchorEl] = useState<HTMLButtonElement | null>(null);
+
+    const contentScope = useContentScope();
+    const apolloClient = useApolloClient();
+    const dependencyMap = useDependenciesConfig();
+
+    const handleMenuClose = () => {
+        setAnchorEl(null);
+    };
+
+    const damFile = input.value;
+
+    if (damFile) {
+        const showMenu = Boolean(dependencyMap["DamFile"]);
+        return (
+            <>
+                <AdminComponentPaper disablePadding>
+                    <Grid container alignItems="center" spacing={3}>
+                        <Grid item xs>
+                            <Typography variant="subtitle1">{damFile.name}</Typography>
+                            <Typography variant="body1" color="textSecondary">
+                                <DamPathLazy fileId={damFile.id} />
+                            </Typography>
+                        </Grid>
+                        {showMenu && (
+                            <Grid item>
+                                <IconButton
+                                    onMouseDown={(event) => event.stopPropagation()}
+                                    onClick={(event) => {
+                                        event.stopPropagation();
+                                        setAnchorEl(event.currentTarget);
+                                    }}
+                                    size="large"
+                                >
+                                    <MoreVertical />
+                                </IconButton>
+                            </Grid>
+                        )}
+                    </Grid>
+                    <Divider />
+                    <AdminComponentButton startIcon={<Delete />} onClick={() => input.onChange(undefined)}>
+                        <FormattedMessage id="comet.form.file.empty" defaultMessage="Empty" />
+                    </AdminComponentButton>
+                </AdminComponentPaper>
+                {showMenu && (
+                    <Menu anchorEl={anchorEl} keepMounted open={Boolean(anchorEl)} onClose={handleMenuClose}>
+                        <MenuItem
+                            onClick={async () => {
+                                // id is checked three lines above
+                                // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
```

Comment:
i think this no-non-null-assertion is not needed.

Reply:
maybe a check if the `DamFile` entry exists in the map is necessary?

Reply:
Done https://github.com/vivid-planet/comet/pull/3988/commits/c7108fb44ca7aa4f8e798660aa2907af8c7f274c